composer/core/state.py

```diff
@@ -132,6 +147,32 @@ def fsdp_get_optim_state_dict(model: torch.nn.Module,
         raise NotImplementedError(f'No valid FSDP state_dict_type for {state_dict_type}')
 
 
+def _legacy_optim_state_dict_to_load(optim_state_dict: Dict[str, Any],
+                                     model: torch.nn.Module,
+                                     optim: torch.optim.Optimizer,
+                                     state_dict_type: str = 'full'):
```

Comment:
```suggestion
                                     state_dict_type: str = 'full',):
```
nit. I'll add a lint for this soon...